rpi5.yaml

```diff
@@ -428,3 +428,27 @@ parameters:
                   "%{SOC_FAMILY}-%{MACHINE}-can-%{DOMD_CAN_TYPE}.dtbo": "%{YOCTOS_WORK_DIR}/%{DOMD_BUILD_DIR}/tmp/deploy/images/%{MACHINE}/%{SOC_FAMILY}-%{MACHINE}-can-%{DOMD_CAN_TYPE}.dtbo"
     no:
       default: true
+
+  ENABLE_HDMI:
+    desc: "Allow HDMI in DomD"
+    yes:
+      overrides:
+        components:
+          domd:
+            builder:
+              conf:
+                - [MACHINE_FEATURES:append, " domd_hdmi"]
+                - [DOMD_OVERLAYS:append, " %{SOC_FAMILY}-%{MACHINE}-hdmi.dtbo"]
+                - [DISTRO_FEATURES:append, "opengl"]
```

Comment:
I would recommend replacing `"opengl"` with `" opengl"` here.
But a more important question is "Do we need to enable opengl to have hdmi working?"